### PR TITLE
feat(helm): add setting to disable service account creation

### PR DESCRIPTION
**Pull request by MaxTNielsen**
Add a setting to disable service account creation

### CODE REVIEW

**Comment by MaxTNielsen** [comment]
I have read the CLA Document and I hereby sign the CLA

File: helm/coder/values.yaml [hunk 1]
```diff
@@ -113,6 +113,8 @@ coder:
     annotations: {}
     # coder.serviceAccount.name -- The service account name
     name: coder
+    # coder.serviceAccount.name -- Whether to create the service account or use existing service account
+    create: true
```
Comment (kylecarbs) [inline helm/coder/values.yaml:117]: I think defaulting to false is better in this scenario. So maybe "disableCreate" or something?

Curious what @ericpaulsen thinks

Comment (ericpaulsen) [reply helm/coder/values.yaml]: i agree - defaulting to `false` is best.

**Review by ericpaulsen — APPROVED** [review]

**Comment by johnstcn** [comment]
@ericpaulsen Let's make sure there's a corresponding test case for the Helm chart!

**Comment by johnstcn** [comment]
@sreya see https://github.com/coder/coder/pull/15197, I want to keep the same default behaviour

**Pull request merged**